litmus-portal/frontend/src/components/YamlEditor/Editor.tsx

```diff
@@ -56,23 +55,13 @@ const YamlEditor: React.FC<YamlEditorProps> = ({
   readOnly,
   setButtonState,
   saveWorkflowChange,
-  className,
 }) => {
   const classes = useStyles();
   const { palette } = useTheme();
 
-  const { t } = useTranslation();
-
   const [isValid, setIsValid] = useState(true);
 
-  const [errors, setErrors] = useState({
```

Comment:
Without the line number, how are we showing, which line is having error?

Reply:
Ace Editor handles this behaviour by default, it adds a red cross and on hover shows the exact issue. 

Here is a screenshot for reference: 
![editorErrorHover](https://user-images.githubusercontent.com/34975209/120428081-a708a280-c390-11eb-805f-6ae4757b2ba7.png)